src/lib/markbind/src/lib/markdown-it/index.js

```diff
@@ -109,8 +109,10 @@ markdownIt.renderer.rules.fence = (tokens, idx, options, env, slf) => {
   const heading = token.attrGet('heading');
   const codeBlockContent = `<pre><code ${slf.renderAttrs(token)}>${str}</code></pre>`;
   if (heading) {
-    return '<div class="code-block">'
-      + `<div class="code-block-heading"><span>${heading}</span></div>`
+    const renderedHeading = markdownIt.renderInline(heading);
+    const headingStyle = (renderedHeading === heading) ? 'code-block-heading' : 'code-block-heading inline-markdown-heading';
+      return '<div class="code-block">'
```

Comment:
indent here should be 2 to the left, is github displaying it wrongly?

Reply:
No you're right. The pains of not having eslint consider this file. 😞